### ISSUE TITLE
[Feature]: Add tenor.com as default to the WaterFrames list of whitelisted urls

### ISSUE THREAD

**Issue by IamKahina** [ThePansmith/Monifactory#1183]
### Suggestion

It is a massive platform of images and gifs, so it makes sense to me for it to be whitelisted by default in WaterFrames as you often have to upload a gif from there to somewhere else to be able to display it.

**Comment by tomprince** [comment]
I don't think this modpack wants to be in the business of allowlisting image sites, so you would be better off reporting this to the upstream mod, or just changing the config in your save/on your server.

(I will note that tenor.co (with a missing `m`) *is* whitelisted by default, so I suspect that they just need to update to the newer domain name)

**Comment by Xefyr0** [comment]
Resolved in 077a7e6873717f277feecf78c0fc99ed2fb5ccb3.
update to Waterframes/Watermedia adds Tenor to the mod's built-in whitelist.
